hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheProxyBase.java

```diff
@@ -61,6 +62,12 @@
 
     private static final int TIMEOUT = 10;
 
+    /**
+     * Speculative factor to be used when initialising collections
+     * of an approximate final size.
+     */
+    private static final double SIZING_FUDGE_FACTOR = 1.3;
```

Comment:
I think we have this factor and formula three times in the code now. Should we put this into one of the utility classes?